packages/styles/scss/components/content-switcher/_content-switcher.scss

```diff
@@ -52,15 +52,33 @@
     color: $text-secondary;
     text-align: left;
     text-decoration: none;
-    transition: all $duration-fast-01 motion(standard, productive);
+    transition: all $duration-moderate-01 motion(standard, productive);
     white-space: nowrap;
 
+    &::after:not(&:disabled) {
```

Comment:
```suggestion
    &::after {
```

Reply:
To cover the case of a disabled content switcher, add a new rule like so:

```scss
&:disabled::after {
  display: none;
}
```

Reply:
I can try these changes, but it was working before this way, nothing in carbon's elements has changed so I don't know why the changes can't be seen in storybook. (I think carbon changed the v10 and v11 styles and which packages pertain to which)

Reply:
From what I can see the styles are loaded in correctly, but the selector `&::after:not(&:disabled)` doesn't match. Changing it to `&:not(&:disabled)::after` works but is higher in specificity than `&::after` on line 200 meaning the `scaleY(1)` will not be shown. Hence I suggested to simplify the first selector and create another to address the disabled state.

I'm not 100% sure but the first selector might just be invalid since it would query the `after` pseudo element not being the disabled parent. Did you use this exact selector before?

Reply:
I did use that exact same selector and worked fine, that's why i found it weird. But thanks to your suggestions it works now